nexus/pkg/server/clients.go

```diff
@@ -28,8 +28,8 @@ func (t *authedTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	return t.wrapped.RoundTrip(req)
```

Comment:
eventually we probably want to organize this differently, but i think it's ok for now

Reply:
yes.  client changes are needed for sure.